modules/launcher/src/main/java/org/mule/module/launcher/MuleSharedDomainClassLoader.java

```diff
@@ -84,6 +84,15 @@ public MuleSharedDomainClassLoader(String domain, ClassLoader parent)
         }
     }
 
+    /**
+     * For testing purposes
+     */
+    protected MuleSharedDomainClassLoader(ClassLoader parent)
```

Comment:
Adding things "for testing purposes" is not a good idea, even worst when you add a constructor that creates an incomplete/invalid instance